bubble_sort.rb

```diff
@@ -0,0 +1,49 @@
+def is_sorted(ar)
+  length_of_array = ar.length
+  i = 0
+  while i < length_of_array do
+    current_element = ar[i]
+    next_element = ar[i]
+    if i != length_of_array - 1
+      next_element = ar[i + 1]
+    end
```

Comment:
Lines 6 to 9 can be simplified to `next_element = ar[i+1] if i < length_of_array`